src/components/questions/modals/AnswerForm.js

```diff
@@ -0,0 +1,67 @@
+import React, { useState } from "react";
+import Button from '@material-ui/core/Button';
+import { TextField } from '@material-ui/core';
+import apiController from '../../../apiController';
+import './form.css';
+
+var AnswerForm = (props) => {
+  const [answer, setAnswer] = useState("");
+  const [nickname, setNickname] = useState("");
+  const [email, setEmail] = useState("");
+
+  var questionId = props.questionId;
+
+  var handleSubmit = (event) => {
+    event.preventDefault();
+    console.log('form submitted ');
+    props.handleClose();
+
+    var params = {
+      body: answer,
+      name: nickname,
+      email: email,
+      photos: []
+    }
+    apiController.postAnswer(questionId, params);
+  }
+
+  return (
+    <form onSubmit = {handleSubmit} autoComplete="off">
+      <div className = 'form-container'>
+       <TextField
+        required id="standard-required"
```

Comment:
I can't find any documentation on this `standard-required` id. Do you know where you found it/why this one is `standard-required` while the others are `standard-basic`?